data/reports/m1_proposal.md

```diff
@@ -0,0 +1,41 @@
+### Section 1: Motivation and purpose
+
+Our role: Public Health Organization
+
+Target Audience: NGO, Policymakers, Public Health Organizations
+
+Tuberculosis (TB) is a pervasive infectious disease primarily affecting the lungs and stands as the principal cause of mortality among individuals with HIV. Annually, it afflicts approximately 10 million individuals globally, leading to around 1.5 million deaths (WHO, 2019). To curb the spread of TB and mitigate its impact, it necessitates strategic allocation of medical resources and medications by non-governmental organizations (NGOs), given the current global prevalence of this disease (TB Alliance, n.d.). Despite the urgency, NGOs frequently encounter obstacles in obtaining timely, detailed, and comprehensive data on TB incidence, treatment outcomes, and main risk factors. The process of gathering, processing, and analyzing such data is often fraught with challenges, including time constraints and a lack of necessary programming skills, which impedes efficient and effective data utilization. To bridge this gap, we envision the development of a data visualization application specifically designed to empower NGOs and public health entities. This tool will provide an intuitive interface to explore the global dynamics of tuberculosis, monitoring its prevalence and trends over time across different nations. Moreover, it will offer insights into critical risk factors, including age, HIV status, and gender, thereby equipping NGOs with the necessary knowledge to optimize their resource distribution and intervention strategies. Our app generally aims to streamline the decision-making process for health professionals and policy-makers, fostering more targeted and impactful responses to the global TB crisis.
+
+### Section 2: Data Description
+
+We will be developing a dashboard to visualize the **global prevalence of Tuberculosis (TB) over time** and the **associated risk factors per country**.
+
+We are using two datasets from the World Health Organization (WHO) which can be found [here](https://www.google.com/url?q=https://extranet.who.int/tme/generateCSV.asp?ds%3Destimates&sa=D&source=docs&ust=1711469479701182&usg=AOvVaw2Uui8IYP7fyZ3E0_nXPnGw) and [here](https://www.google.com/url?q=https://extranet.who.int/tme/generateCSV.asp?ds%3Destimates_age_sex&sa=D&source=docs&ust=1711469600536959&usg=AOvVaw3O2Ts9QUTKv5xk08Arn6fE). The first dataset summarizes global trends of TB for 217 countries from 2000 to 2022.
+
+-   Geographic and demographic information: `iso2 code`
+
+-   Annual data per country (identified by `year`):
+
+    -   Population (`e_pop_num`),
+
+    -   WHO-generated estimates:
```

Comment:
Maybe add a section on best vs. upper and lower estimates if we're going to use those for confidence intervals